### PR TITLE
Newteam

### PR DESCRIPTION
**Pull request by yug105**
deleted the blog editor files and team 2027 changes updated

### CODE REVIEW

File: components/Members/Admin/dashboard.tsx [hunk 1]
```diff
@@ -112,6 +112,15 @@ const Dashboard = () => {
                                 <h5 className="text-2xl text-center font-bold mt-2" style={styles.textSecondary}>Tasks</h5>
                             </div>
                         </Link>
+                        {/* <Link href="/admin/createblogs">
```
Comment (priyansh210) [inline components/Members/Admin/dashboard.tsx:115]: Dont comment out the componenet, remove if not part of the PR

File: components/Members/sidenav.tsx [hunk 2]
```diff
@@ -150,6 +150,28 @@ const SideNav = () => {
                  Hackathon Registrations
                 </p>
               </Link>
+              {/* <Link href="/admin/createblogs">
```
Comment (priyansh210) [inline components/Members/sidenav.tsx:153]: Same as above

File: components/Team/team.jsx [hunk 3]
```diff
@@ -5,7 +5,9 @@ import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import {faFacebook, faTwitter, faInstagram, faLinkedin, faGithub,  faWordpress} from '@fortawesome/free-brands-svg-icons';
 import team from "../../images/team.jpeg"
 import RajeshSir from "../../images/rajeshsir.webp"
-import { founders, year2025, year2026 } from "../../constants/members";
+
+import { founders, year2024, year2025, year2026,year2027 } from "../../constants/members";
```
Comment (priyansh210) [inline components/Team/team.jsx:9]: Fix formatting

**Review by priyansh210 — COMMENTED** [review]
Make the necessary modifications

File: constants/members.ts [hunk 4]
```diff
@@ -1375,7 +1387,9 @@ export const year2024: Member[] = [
   {
     name: "DIVYANSH GARG",
     image: divyansh,
+
     desc: "SWE at Care Health",
```
Comment (rupesh031) [inline constants/members.ts:1391]: Works at CRED

**Comment by MashaalSayeed** [comment]
Change the PR to the team-2027 branch

File: constants/members.ts [hunk 5]
```diff
@@ -1649,3 +1667,115 @@ export const year2026: Member[] = [
     facebook: "",
   }
 ];
+export const year2027: Member[] = [
```
Comment (rupesh031) [inline constants/members.ts:1670]: Add Links to available profiles
